### DISCUSSION TITLE
Gavilya v2.9.0.2209-pre1

### DISCUSSION THREAD

**Discussion by lpeyr** [Leo-Corporation/Gavilya#282] [Releases]
# What's new?
### New
- 2be9b4212088799d0b0bab21aa1a5ba6a4cb2742 Added FPS Counter (#278) (From [DareathX](https://github.com/DareathX) / [CustomFpsCounter](https://github.com/DareathX/CustomFpsCounter)) + #276
- 62bb014eaf140a2fac4c11c7239fae7cdda87e78 Added the possibility to show/hide the FPS Counter (#279)

### Fixed
- db8f4644857e759444975123a1d18e8e3efa427d Fixed an issue when selecting a cover
- 9b0213501a9aea10b1b8d9b80fe7c228a703245d Fixed topmost issues

### Updated
- 8bf09c55b667487a57e264cf78acae165d13beb1 Updated API Key

## Known issues
[Click here](https://github.com/Leo-Corporation/Gavilya/issues?q=is%3Aopen+is%3Aissue+label%3Abug) to see all the know issues and bugs.

## Files
- GavilyaSetup.exe
## Download
[Click here](http://www.mediafire.com/file/o17u9ejfc3po061/GavilyaSetup.exe/file) to download Gavilya.

<hr /><em>This discussion was created from the release <a href='https://github.com/Leo-Corporation/Gavilya/releases/tag/2.9.0.2209-pre1'>Gavilya v2.9.0.2209-pre1</a>.</em>

**Comment by sgamesdev** [comment]
I'll try to crack the encryption method  and tell you how i did it so you can improve it
